### DISCUSSION TITLE
Feat: Shadow around the box

### DISCUSSION THREAD

**Discussion by iflamed** [tailwindlabs/tailwindcss#12386] [Ideas]
Current, all shadow class are set the shadow downside to the box. No way to set the shadow to around the box.

**Comment by kaelansmith** [comment]
@iflamed you can either:

1. override/extend the `theme.boxShadow` property to include shadow presets that have different x&y offset values,
2. use arbitrary values syntax, eg. `shadow-[10px_35px_60px_-15px_rgba(0,0,0,0.3)]`, or
3. use a plugin such as the one I built, [Tailwind Extended Shadows](https://github.com/kaelansmith/tailwind-extended-shadows), which provides extra shadow utility classes that give you fine-grain control over shadow x&y offset, blur, spread, and opacity; eg. `class="shadow shadow-x-2 shadow-y-3 shadow-blur-4 -shadow-spread-0.5 shadow-opacity-10"`. It even adds utilities to auto-generate multiple shadow layers; eg. `class="shadow-sm shadows-4 shadows-scale-2"`
